brave/brave-core
# feat(wallet): Move Account Actions into Menu

## Description 
- Moved the `Account` actions into a `More Menu`
- Moved the `WalletSettingsMenu` in the `wallet-menus` directory.


Resolves <https://github.com/brave/brave-browser/issues/29954>



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
1. Go to the `Accounts` tab
2. Click on the `More Menu` for an account
3. All actions should work as expected.

https://user-images.githubusercontent.com/40611140/234097639-d3f1438d-e4c4-4b8d-9538-08b7a1ea1997.mov
